compiler/src/iree/compiler/Dialect/Encoding/IR/EncodingAttrs.cpp

```diff
@@ -21,42 +21,120 @@
 
 namespace mlir::iree_compiler::IREE::Encoding {
 
+//===---------------------------------------------------------------------===//
+// encoding.encoding
+//===---------------------------------------------------------------------===//
+
+static AffineMap getComposedAffineMap(Attribute attr) {
```

Comment:
Please add a doc to the method because the `attr` could be either AffineMapAttr or ArrayAttr. It is not clear what the method is returned when the input type is `Attribute`. We also return `AffineMap()` when it is not an expected input.

Reply:
Done.